opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.cpp

```diff
@@ -125,9 +125,10 @@ namespace Opm
             int wellStateStepNumber = ( ! substep && timer.reportStepNum() > 0) ?
                 (timer.reportStepNum() - 1) : timer.reportStepNum();
             // collect all solutions to I/O rank
-            isIORank = parallelOutput_->collectToIORank( localState, localWellState, sol, wellStateStepNumber );
+            isIORank = parallelOutput_->collectToIORank( localState, localWellState, localCellData, wellStateStepNumber );
         }
 
+        const data::Solution& cellData = ( parallelOutput_ && parallelOutput_->isParallel() ) ? parallelOutput_->globalCellData() : localCellData;
```

Comment:
it would probably be more transparent if `collectToIORank()` was called unconditionally with it being a no-op in sequential mode. (just like `Dune::CollectiveCommunication`)